docs/src/Utilities/Microphysics.md

```diff
@@ -0,0 +1,229 @@
+# Microphysics Module
+
+The `Microphysics` module provides 1-moment bulk parameterization of cloud microphysical processes. The module describes the warm rain (no ice and snow) formation and is based on the ideas of Kessler 1995. The cloud microphysics variables are expressed as specific humidities:
+  - q_tot - total water specific humidity
+  - q_vap - water vapor specific humidity
+  - q_liq - liquid water specific humidity
+  - q_rai - rain water specific humidity
+Constant parameters used in the parameterization are defined in `MicrophysicsParameters` module.
+
+Parameterized processes include
+  - rain sedimentation with mass weighted average terminal velocity
+  - condensation/evaporation of cloud water
+  - autoconversion
+  - accretion
+  - evaporation of rain water
+
+## terminal velocity
+
+The mass weighted average rain terminal velocity is calculated assuming Marshall-Palmer distribution of rain drops.
+
+The assumed rain-drop size distribution (Marshall Palmer 1948 eq. 1) as a function of rain drop diameter:
+```math
+\begin{equation}
+n(D) = n_{0_{MP}} exp\left(- \lambda_{MP} D \right)
+\end{equation}
+```
+where
+ - ``D`` is the drop diameter
+ - ``n_{0_{MP}}`` and ``\lambda_{MP}`` are the Marshall-Palmer distribution parameters.
+
+In the remaining part of the Microphysics module derivation this size distribution is expressed as a function of drop radius ``r``
+```math
+\begin{equation}
+n(D) = 2 n_{0_{MP}} exp\left(- 2 \lambda_{MP} r \right)
+\end{equation}
+```
```

Comment:
My preference would be just to write the equation in terms of radius, and absorb the 2's into the constants n_0 and \lambda. No need to state the equation with diameter if we do not use it (it just confuses).